rpb/riak_dt/riak_dt.pb.go

```diff
@@ -75,6 +81,7 @@ func (x *MapField_MapFieldType) UnmarshalJSON(data []byte) error {
 	*x = MapField_MapFieldType(value)
 	return nil
 }
+func (MapField_MapFieldType) EnumDescriptor() ([]byte, []int) { return fileDescriptor0, []int{0, 0} }
```

Comment:
Good question. @christophermancini what's the output of this command on your system?

```
protoc --version
```